exporters/prometheus/config.go

```diff
@@ -132,9 +148,11 @@ func WithoutScopeInfo() Option {
 	})
 }
 
-// WithNamespace configures the Exporter to prefix metric with the given namespace.
-// Metadata metrics such as target_info are not prefixed since these
-// have special behavior based on their name.
+// WithNamespace configures the Exporter to prefix metric with the given
+// namespace. Metadata metrics such as target_info are not prefixed since these
+// have special behavior based on their name. Namespaces will be prepended even
+// if NoTranslation is set as a translation strategy. An empty string namespace
```

Comment:
What does `An empty string namespace disables prepending` mean in this case? Isn't it just a no-op since the default is not to append a namespace?